frontend/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:18-alpine AS build
+
+WORKDIR /app
+COPY package.json package-lock.json ./
+RUN npm install
+COPY ..
```

Hadolint (2.12.0)

[error] unexpected end of line. At least two arguments are required for COPY

(DL1000)